source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -567,12 +572,10 @@ If the pool is "closed" or "paused", any attempt to check out a `Connection
 being "paused" MUST be considered a retryable error and MUST NOT be an error
 that marks the SDAM state unknown.
 
-If minPoolSize is set, the `Connection <#connection>`_ Pool MUST have at least
-minPoolSize total `Connections <#connection>`_ while it is "ready". If the pool does
-not implement a background thread, the checkOut method is responsible for
-`populating the pool
-<#populating-the-pool-with-a-connection-internal-implementation>`_ with enough
-`Connections <#connection>`_ such that this requirement is met.
+If the pool does not implement a background thread, the checkOut method is
+responsible for ensuring that the pool is `populated
+<#populating-the-pool-with-a-connection-internal-implementation>`_ with at least minPoolSize
+`Connections <#connection>`_.
```

Comment:
This duplicates what was said in the "Populating the Pool with a Connection (Internal Implementation)" section. On one hand duplication is beneficial to the reader, on the other hand it is detrimental to the maintainability of the spec. In this case I failed to argue myself into preferring one or another approach, even though initially the duplication rose a flag for me.